src/components/Notifications/NotificationItemNFTLink.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import { NotificationItemNFTLinkProps } from "./NotificationItemNFTLink.types"
+import { Link } from "./NotificationItemNFTLink.styled"
+
+const NotificationItemNFTLink = React.memo(
+  (props: NotificationItemNFTLinkProps) => {
+    const { link, rarity, name } = props
+    return (
+      <Link href={link} rarity={rarity}>
```

Comment:
If this is a link, should it have a property to identify how it should be open (on a new tab, on the same window, etc)?

Reply:
the default target of the Link component is `_self`, do you think we should add the target when we use the default one?